tests/chmod/11.t

```diff
@@ -63,6 +63,12 @@ for type in regular fifo block char socket symlink; do
 		create_file ${type} ${n1} 0640 65534 65534
 		expect 0 symlink ${n1} ${n2}
 		case "${os}" in
+		Darwin)
```

Comment:
Sidenote: I still need to run the tests with OS X Sierra (10.12.x). I would assume that they "just work", but I'm not willing to bank on that 100%.